lib/board.rb

```diff
@@ -1,10 +1,13 @@
+require_relative 'express_board_in_rows'
+
 class Board
+  include BoardAsRows
   DENOTE_EMPTY = '-'
-  attr_reader :board, :num_of_blank_spaces, :blank_positions
+  attr_reader :board, :num_of_played_positions, :blank_positions
 
   def initialize(size)
     @board = Array.new(size) { Array.new(size) { DENOTE_EMPTY } }
-    @num_of_blank_spaces = size * size
+    @num_of_played_positions = 0
     @blank_positions = make_free_positions_hash(size.times.to_a.product(size.times.to_a))
```

Comment:
It looks like `empty`, `blank`, and `free` may all mean the same thing. Let's pick one and be consistent throughout the code base.

Reply:
Resolved at 5fbd8db67c30a860f9e237172e7fd1152cb36f01

Reply:
Resolved at 9dbbabed8c5453b180590dc36e5ca3e44f5c06c8

Reply:
Resolved at 2d157729000732e9b53f581d1f5213a341c3fed0